pyproject.toml

```diff
@@ -0,0 +1,125 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mozilla-django-oidc-db"
+version = "0.18.1"
+description = "A database-backed configuration for mozilla-django-oidc"
+authors = [
+    {name = "Maykin Media", email = "support@maykinmedia.nl"}
+]
+readme = "README.rst"
+license = {file = "LICENSE"}
+keywords = ["OIDC", "django", "database", "authentication"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: Django",
+    "Framework :: Django :: 4.2",
+    "Intended Audience :: Developers",
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+requires-python = ">=3.10"
+dependencies = [
+    "django>=4.2",
+    "django-jsonform",
+    "django-solo",
+    "glom",
+    "mozilla-django-oidc>=3.0.0",
+    "typing-extensions>=4.0.0",
+]
+
+[project.urls]
+Homepage = "https://github.com/maykinmedia/mozilla-django-oidc-db"
+Documentation = "https://mozilla-django-oidc-db.readthedocs.io/en/latest/"
+"Bug Tracker" = "https://github.com/maykinmedia/mozilla-django-oidc-db/issues"
+"Source Code" = "https://github.com/maykinmedia/mozilla-django-oidc-db"
+Changelog = "https://github.com/maykinmedia/mozilla-django-oidc-db/blob/master/CHANGELOG.rst"
+
+[project.optional-dependencies]
+tests = [
+    "psycopg2",
+    "pytest",
+    "pytest-django",
+    "pytest-mock",
+    "pytest-recording",
+    "requests-mock",
+    "factory-boy",
+    "pyquery",
+    "tox",
+    "isort",
+    "black",
+    "flake8",
+]
+coverage = [
+    "pytest-cov",
+]
+docs = [
+    "sphinx",
+    "sphinx-rtd-theme",
+]
+release = [
+    "bump-my-version",
+    "twine",
+]
+
+[tool.setuptools.packages.find]
+include = ["mozilla_django_oidc_db*"]
+namespaces = false
+
+[tool.isort]
+profile = "black"
+combine_as_imports = true
+known_django = "django"
+known_first_party="mozilla_django_oidc_db"
+sections=["FUTURE", "STDLIB", "DJANGO", "THIRDPARTY", "FIRSTPARTY", "LOCALFOLDER"]
```

Comment:
probably we need to add the `skip_glob` for `.tox` and `env` here too, it really slows down things a lot on local dev machines